packages/camera/example/android/app/build.gradle

```diff
@@ -57,6 +57,10 @@ flutter {
 }
 
 dependencies {
+    // TODO: remove these dependencies when engine POM is working correctly.
+    implementation 'androidx.lifecycle:lifecycle-runtime:2.1.0'
```

Comment:
What happens if there's another plugin that requires an incompatible version of lifecycle (e.g 1.1.1) before having the POM?
What's the timeline for the POM?

cc @blasten

Reply:
You don't need these dependencies. @matthew-carroll